test/e2e/storage/vsphere/persistent_volumes-vsphere.go

```diff
@@ -178,7 +178,7 @@ var _ = utils.SIGDescribe("PersistentVolumes:vsphere", func() {
 		3. Verify that written file is accessible after kubelet restart
 	*/
 	It("should test that a file written to the vspehre volume mount before kubelet restart can be read after restart [Disruptive]", func() {
-		utils.TestKubeletRestartsAndRestoresMount(c, f, clientPod, pvc)
```

Comment:
so all the functions are not using pvc at all before?

Reply:
I think originally the test functions created the PVCs inside.  But when it got refactored, the PVC creation got moved outside.  So the PVC argument is not used in any of these test functions anymore.